Cargo.toml

```
@@ -13,9 +13,8 @@ keywords = [ "vibrancy", "acrylic", "blur", "tauri", "gui", "plugin", "tauri-plu
 categories = [ "gui" ]
 
 [dependencies]
-tao = { git = "https://github.com/tauri-apps/tao", branch = "next", default-features = false, features = [], optional = true }
-tauri = { git = "https://github.com/tauri-apps/tauri", branch = "next", default-features = false, features = [], optional = true }
-
+tao = { version = "0.6", default-features = false, features = [], optional = true }
+tauri = { version = "1.0.0-rc.0", default-features = false, features = [], optional = true }
```

I am not a semver expert but would this use the current tauri/tso version of the user?
```suggestion
tao = { version = "^0.6", default-features = false, features = [], optional = true }
tauri = { version = "^1", default-features = false, features = [], optional = true }
```

---

Cargo handles version requirements without operators the same as if it would have a caret `^`.
It completely ignores pre-release versions if you don't explicitly add a pre-release suffix tho.

So for tao both versions are the same and the tauri one can't work this way until we release stable.

---

cool, this is fine for now then.